conans/client/userio.py

```diff
@@ -26,14 +27,17 @@ def color_enabled(stream):
 
 def init_colorama(stream):
     import colorama
-
     if not color_enabled(stream):
-        if os.getenv("NO_COLOR") != "1":
-            colorama.init(strip=True)
-        return False
+        if os.getenv("NO_COLOR") == "1":
```

Comment:
I see here: https://no-color.org/ that: _"when present (regardless of its value), prevents the addition of ANSI color._"
So maybe checking that is not `None` is ok.